compiler/qsc_codegen/Cargo.toml

```diff
@@ -9,5 +9,16 @@ edition.workspace = true
 license.workspace = true
 
 [dependencies]
-qsc_ast = { path = "../qsc_ast" }
+num-bigint = { workspace = true }
+num-complex = { workspace = true }
+qsc_eval = { path = "../qsc_eval" }
+qsc_data_structures = { path = "../qsc_data_structures" }
 qsc_frontend = { path = "../qsc_frontend" }
+qsc_fir = { path = "../qsc_fir" }
+qsc_hir = { path = "../qsc_hir" }
+quantum-sparse-sim = { git = "https://github.com/qir-alliance/qir-runner", rev = "e96b9a12a43e98f83f2d4d499f46d651d0c6c341", default-features = false}
```

Comment:
Move to root cargo.toml and ref workspace. This also needs to be done in the qsc_eval trait which is duplicated.

Reply:
Good catch!